libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/AlertDialogContent.kt

```diff
@@ -231,12 +231,11 @@ internal fun AlertDialogContent(
                         Box(
                             // Align the title to the center when an icon is present.
                             Modifier
-                                .then(
-                                    if (applyPaddingToContents)
-                                        Modifier
-                                    else
-                                        Modifier.padding(DialogContentDefaults.externalHorizontalPadding)
-                                )
+                                .apply {
+                                    if (!applyPaddingToContents) {
+                                        then(Modifier.padding(DialogContentDefaults.externalHorizontalPadding))
+                                    }
+                                }
```

Comment:
This does not seem to be equivalent, see the failure in the screenshot test:
<img width="1563" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/32529b52-ec21-4f39-8d8e-2d3dc8ea4619">

Reply:
The other change is because the changed `material.Text` to `material3.Text` and the rendering is slightly different.

Reply:
Actually the returned value of `apply` is its receiver. This is not the case for `then`, which return `other` in this case.
```kotlin
                            // Align the title to the center when an icon is present.
                            (
                                if (applyPaddingToContents)
                                    Modifier
                                else
                                    Modifier.padding(DialogContentDefaults.externalHorizontalPadding)
                                )
                                .padding(DialogContentDefaults.titlePadding)
                                .align(
                                    if (icon == null) {
                                        Alignment.Start
                                    } else {
                                        Alignment.CenterHorizontally
                                    }
                                )
```

would have worked too (I think), hence a bit hard to read.

Reply:
Oh, true 🤦 . Maybe a `.run { ... }` would have worked, but it would look quite weird there.